examples/with-react-with-styles/pages/_app.js

```diff
@@ -0,0 +1,27 @@
+import App from 'next/app'
+import React from 'react'
+import WithStylesContext from 'react-with-styles/lib/WithStylesContext'
+import AphroditeInterface from 'react-with-styles-interface-aphrodite'
+import { DIRECTIONS } from 'react-with-direction'
+
+import defaultTheme from '../defaultTheme'
+
+class MyApp extends App {
```

Comment:
This should use the function version instead: https://nextjs.org/docs/advanced-features/custom-app

Besides that the example looks good to me 👍